.github/workflows/test_ollama.yml

```
@@ -0,0 +1,125 @@
+          GRAPHISTRY_USERNAME: ${{ secrets.GRAPHISTRY_USERNAME }}
+          GRAPHISTRY_PASSWORD: ${{ secrets.GRAPHISTRY_PASSWORD }}
+          LLM_API_KEY: "ollama"
+          LLM_PROVIDER: "ollama"
+          LLM_ENDPOINT: "http://localhost:11434"
+          LLM_MODEL: "ollama/llama3.2"
+          EMBEDDING_PROVIDER: "ollama"
+          EMBEDDING_MODEL: "avr/sfr-embedding-mistral:latest"
+          EMBEDDING_ENDPOINT: "http://localhost:11434/api/embeddings"
+          EMBEDDING_DIMENSIONS: "4096"
+          HUGGINGFACE_TOKENIZER: "Salesforce/SFR-Embedding-Mistral"
+        run: poetry run python ./examples/python/simple_example.py
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)